### ISSUE TITLE
No reward in wallet, grin wallet info is always 0, grin wallet outputs are always Unconfirmed

### ISSUE THREAD

**Issue by blabarna** [mimblewimble/grin#1316]
Good day!
        I have run grin and grin-mainer successfully during last day. The mining speed is up to 7GPS
        But when I run grin wallet info (with password), it is always empty(42 Grin I have got with command curl gringod.info)
____ Wallet Summary Info as of 35370 ____
Total                            | 42.170000000
Awaiting Confirmation            | 0.000000000
Immature Coinbase                | 0.000000000
Currently Spendable              | 42.170000000
---------                        | ---------
(Locked by previous transaction) | 0.000000000
        The hardware are:  2 x GeForce GTX 1070
        Both, grin server and miner works on default ports (from grin.toml), which are open by \
outside.
tcp        0      0 127.0.0.1:13413         0.0.0.0:*               LISTEN      25895/grin
tcp        0      0 0.0.0.0:13414           0.0.0.0:*               LISTEN      25895/grin
tcp        0      0 0.0.0.0:13415           0.0.0.0:*               LISTEN      25839/grin
tcp        0      0 127.0.0.1:13416         0.0.0.0:*               LISTEN      25895/grin
tcp        0      0 127.0.0.1:13420         0.0.0.0:*               LISTEN      25895/grin
       Options, which was changed in grin.toml:
                 [mining_server]
                 burn_reward = false #was yet set
                 [wallet]
                 api_listen_interface = "0.0.0.0"
                 [logging]
                 stdout_log_level = "Debug"

        The consequnce command of run
            grin wallet -e listen -p "Password"
            grin server run
            grin-miner
       I use profile "debug" on grin-miner with switched on cuda-mining (type_filter = "cuda") and on grin server
        I had read the manual https://www.grin-forum.org/t/how-to-mine-cuckoo-30-in-grin-help-us-test-and-collect-stats/152/8
        I used the last release of grin & grin-miner from github and clear my previous .grin folder in home directory
        And try new installation in various folders twice
        The problem is the same

        Also the issue was found which seems like mine case:
        https://github.com/mimblewimble/grin/issues/143
        (the problem was - wallet.dat is empty and all outputs as "Unconfirmed" - in grin walle\
t info)

        But recomendations from issue does not help me

        I had wrote to gitter (by my nick blabarna) and community members gave me a few advices
        https://gitter.im/grin_community/Lobby
        The result always is the same.

        Please, help me to mine the grins in my wallet!
        I have attached both log files
[grin.log.zip](https://github.com/mimblewimble/grin/files/2259403/grin.log.zip)
[grin-miner.log.zip](https://github.com/mimblewimble/grin/files/2259405/grin-miner.log.zip)

Thank you

**Comment by hashmap** [comment]
I checked logs, it looks like your miner failed to find any solutions at all which is extremely unlikely for 2 gpu setup. 
I found this statement confusing:
> burn_reward = false #was yet set

it's `false` for months if I'm not mistaken. It seems you might be running with old configs. Check if you have `hash_header = false` in your `grin-miner.toml`. Btw you can run grin without any arguments, just `grin`. I'd the following:

1. Build the latest version of `grin` and `grin-miner` with default configs from git.

2. Only if you will recieve grins outside open the port in `grin.toml`:
```
[wallet]
api_listen_interface = "0.0.0.0"
```

**Comment by blabarna** [comment]
Thank you very much for your reseach of my logs and your help!
It is very important for me to get any advice in my situation.
I really do not know what is going on.

> burn_reward = false #was yet set
Yes, it is from default config. I had pulled the last master branch from git on 4th of August

> Check if you have hash_header = false in your grin-miner.toml
Yes, I have it

> Build the latest version of grin and grin-miner with default configs from git
You are right. Yesterday I have reinstall rust & cargo and very accurately built the last release of grin & grin-miner from the 6-th of August. I have read again the forum & wiki both about how to build grin, grin-miner

Yesterday I had tried to mine with type_filter = mean_compat_cpu with the default config (but with parameter

> api_listen_interface = "0.0.0.0"

). Result was the same after 1 hour of mining. 
Ok, have set type_filter = cuda and have mined all night through. This did not change the result, my wallet is empty, as before(

Now, I have returned the default value of api_listen_interface = "127.0.0.1" and restart grin & grin-miner. 
I think, it is useful to compare my log files with log file from community members!
(I wish to make the analysis of logs and write my thoughts here)
I have the problem and want to solve it!
